modules/aws/ecr_test.go

```diff
@@ -0,0 +1,23 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
```

Comment:
NIT: If you decide to add a custom error for the non-existent ECR repo, consider adding a unit test that checks for that error type when you call `GetECRRepoE` with a random name.

Reply:
See above comment on the issue with testing for this condition.